### PR TITLE
Fix primitive arrays

### PR DESCRIPTION
**Pull request by Bugagazavr**
Should fixes this https://github.com/ruby-grape/grape-swagger-entity/issues/6

Need test with grape-swagger

### CODE REVIEW

**Comment by dblock** [comment]
Looks good, needs CHANGELOG.

**Comment by Bugagazavr** [comment]
@dblock done need merge both

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,44 +1,96 @@
...
+
+# Offense count: 2
+Style/DoubleNegation:
   Exclude:
-    - 'spec/**/*'
-    - 'test/**/*'
-    - 'lib/grape-swagger/entity.rb'
-    - 'lib/grape-swagger/entity/parser.rb'
+    - 'grape-swagger/example/api/endpoints.rb'
+    - 'graper-swagger/example/api/endpoints.rb'
```
Comment (LeFnord) [inline .rubocop_todo.yml]: [typo] ?

File: lib/grape-swagger/entity/parser.rb [hunk 2]
```diff
@@ -65,7 +65,15 @@ def parse_grape_entity_params(params)
             if entity_options[:values] && entity_options[:values].is_a?(Array)
               memo[entity_name][:enum] = entity_options[:values]
             end
+
```
Comment (LeFnord) [inline lib/grape-swagger/entity/parser.rb]: [note] wow, think this method should be refactored (not in this PR)

**Comment by LeFnord** [comment]
want you make a new release of it to reference in grape-swagger?

File: .rubocop_todo.yml (hunk 1 above)
Comment (Bugagazavr) [inline .rubocop_todo.yml]: thanks

**Comment by Bugagazavr** [comment]
@LeFnord yes

**Comment by dblock** [comment]
@LeFnord and @Bugagazavr you're such a great team!

**Pull request merged**